inferlo/pairwise/optimization/convex_bounds.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+import cvxpy as cp
+
+if TYPE_CHECKING:
+    from inferlo import PairWiseFiniteModel
+
+from inferlo.pairwise.testing import grid_potts_model, tree_potts_model, \
+    line_potts_model
+from inferlo.pairwise.optimization.path_dp import max_lh_path_dp
+
+def LPrelaxation(model: PairWiseFiniteModel):# -> np.ndarray:
+    """Max Likelihood for pairwise model by solving LP relaxation.
+
```

Comment:
Please add reference to this function in docs/api.rst (in "Optimization algorithms" section).
This will automatically add this pydoc to documentation, so it will be visible on ths page:
https://inferlo.readthedocs.io/en/latest/api.html

Before submitting, please make sure it appears in docs and looks nice, especially numbered list.
For that build docs locally:
cd docs
make html

Then open docs/_build/html/api.html